app/scripts/directives/unbindService.js

```diff
@@ -26,11 +26,13 @@
     var serviceInstanceDisplayName = $filter('serviceInstanceDisplayName');
 
     var unbindService = function() {
+      ctrl.deletedBinding = _.first(_.filter(ctrl.bindings, {metadata: {name: ctrl.selectedBinding}}));
+      ctrl.appsForDeletedBinding = angular.copy(ctrl.appsForBinding(ctrl.selectedBinding));
```

Comment:
I believe the copy is unnecessary since the entire map is replaced in the overview controller